apps/nextjs-app/src/features/app/components/field-setting/FieldSetting.tsx

```diff
@@ -265,10 +265,10 @@ const FieldSettingBase = (props: IFieldSettingBase) => {
   return (
     <>
       <Sheet open={visible} onOpenChange={onOpenChange}>
-        <SheetContent className="w-[328px] p-2" side="right">
-          <div className="flex h-full flex-col gap-2">
+        <SheetContent className="w-[400px] p-0" side="right">
```

Comment:
Pay attention to the mobile styles, as there may be issues on small-screen devices.